### ISSUE TITLE
I want to help 

### ISSUE THREAD

**Issue by ashleymorris2** [OpenHogwarts/hogwarts#41]
Looks awesome. Do you you have a roadmap? And need some help on code?

**Comment by adamcyf** [comment]
Are you a mapper ?

**Comment by ashleymorris2** [comment]
Not really. I'm more of a coder, but I have some experience with level design.

**Comment by OpenHogwarts** [comment]
Only a ToDo: https://github.com/OpenHogwarts/hogwarts/wiki/ToDo and mostly is model things related.

¿Is there any missing feature that you would like to add?
Maybe:
- Some sort of minigame like chemistry/potions one? (player must have x sort of items and combine them in a very specific way (that he could read on a book) in order to create a potion)
- Skills system based on character level
- More attack types
- Character customization

**Comment by ashleymorris2** [comment]
I'll have a look and see what I can do

**Comment by HelenasBlog** [comment]
I love harry potter too
